src/quetz_client/cli.py

```diff
@@ -18,10 +21,14 @@ def get_client(*, url=None, token=None) -> QuetzClient:
     token: Optional[str]
         The API key needed to authenticate with the server.
         Defaults to the `QUETZ_API_KEY` environment variable.
+
+    insecure: bool
+        Allow quetz-client to perform "insecure" SSL connections
+        and transfers.
     """
     url = url or os.environ.get("QUETZ_SERVER_URL")
     token = token or os.environ.get("QUETZ_API_KEY")
-    client = QuetzClient.from_token(url, token)
+    client = QuetzClient.from_token(url, token, insecure=insecure)
```

Comment:
Let's not change the client constructor, only the CLI tool.

```suggestion
    client = QuetzClient.from_token(url, token)
    client.session.verify = not insecure
```